go/libraries/doltcore/sqle/dolt_log_table_function.go

```diff
@@ -186,7 +186,7 @@ func (ltf *LogTableFunction) Expressions() []sql.Expression {
 
 // getDoltArgs builds an argument string from sql expressions so that we can
 // later parse the arguments with the same util as the CLI
-func getDoltArgs(ctx *sql.Context, expressions []sql.Expression, functionName string) ([]string, error) {
+func getDoltArgs(ctx *sql.Context, expressions []sql.Expression, Name string) ([]string, error) {
```

Comment:
(minor) lower case "name"?

```suggestion
func getDoltArgs(ctx *sql.Context, expressions []sql.Expression, name string) ([]string, error) {
```